### PR TITLE
Built upload form and did some work on home page

### PR DESCRIPTION
**Pull request by EdwardHoang16**
For the upload form, I'm not sure what data is being sent to which URL, so that needs to filled out still. The basic request template is laid out though. 

For the home page, I still need to add the icons and pictures, but the layout more closely resembles the UI prototype's home page: https://xd.adobe.com/view/83d17ed3-1a34-48a5-4a3f-6ec8886fcf16-f5e0/

### CODE REVIEW

**Review by spcharc — APPROVED** [review]
Looks good. You have finished file upload in the front end, I need to figure out file upload in the backend soon.

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+import React from "react";
 import HomePage from "./HomePage";
 import SignUpPage from "./SignUpPage";
-import axios from "axios";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
 import FarmSearchLocations from "./FarmSearchLocations";
+import { createTheme, ThemeProvider } from "@mui/material";
 
 export default function App() {
     const listOfFarms = [
```
Comment (MichaelRomero99) [inline frontend/src/App.js:9]: As soon as a merge my change, we can start making requests from the DB to get this info.

File: frontend/src/UploadForm.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	function handleFileChange(event) {
+		setSelectedFile(event.target.files[0]);
+		setIsSelected(true);
+	};
+
+	async function handleFileSubmit() {
+		const form = new FormData();
+		form.append('file', selectedFile);
+
+        const response = await axios.post("url", form, {
```
Comment (MichaelRomero99) [inline frontend/src/UploadForm.js:17]: What if the response is not successful?

**Review by MichaelRomero99 — APPROVED** [review]

File: frontend/src/UploadForm.js (hunk 2 above)
Comment (spcharc) [reply frontend/src/UploadForm.js:17]: Just leave it there.

Currently this POST request is dummy implementation and will never work properly (as you see it is sending the request to "url" which is not an actual URL). Once upload functionality in the backend is finished we can complete this part.

**Pull request merged**